### PR TITLE
[docs] update upgrade notes

### PR DESCRIPTION
**Pull request by TomasGottvald**
| Q             | A
| ------------- | ---
|Description, reason for the PR| add better scope to less files, add more info about webline layout task
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: upgrade/UPGRADE-v8.1.0-dev.md [hunk 1]
```diff
@@ -438,28 +438,31 @@ There you can find links to upgrade notes for other versions too.
     ```
 
 - update pages layout to webline layout ([#1464](https://github.com/shopsys/shopsys/pull/1464))
-    - update your custom created pages and wrap them to
-    ```html
+    - if you have your custom design you can skip this task
+    - we have to remove global and restrictive `web__line` and `web__container` and add thes to every single page
+    - update your custom created pages and wrap them to two new div with classes `web__line` and `web__container`
```
Comment (s3tezsky) [inline upgrade/UPGRADE-v8.1.0-dev.md]:
```suggestion
    - update your custom created pages and wrap them into two new `div` with classes `web__line` and `web__container`
```

File: upgrade/UPGRADE-v8.1.0-dev.md [hunk 2]
```diff
@@ -438,28 +438,31 @@ There you can find links to upgrade notes for other versions too.
...
             <div class="web__line">
                 <div class="web__container">
                     ...old content here...
                 </div>
             </div>
         {% endblock %}
-    ```
-    - update default pages according this pull request (https://github.com/shopsys/shopsys/pull/1464/files)
+        ```
+    - update all default pages according this pull request (https://github.com/shopsys/shopsys/pull/1464/files)
```
Comment (s3tezsky) [inline upgrade/UPGRADE-v8.1.0-dev.md]: I would preffer better link to the PR
```suggestion
    - update all default pages according to the [PR #1464](https://github.com/shopsys/shopsys/pull/1464/files)
```

File: upgrade/UPGRADE-v8.1.0-dev.md [hunk 3]
```diff
@@ -659,72 +663,81 @@ There you can find links to upgrade notes for other versions too.
     - add images to
         - `web/assets/frontend/images/custom_checkbox.png`
         - `web/assets/frontend/images/custom_radio.png`
-        
+
         you can [download ours](https://github.com/shopsys/shopsys/blob/8.1/project-base/web/assets/frontend/images/) or use yours
-    - download new images or add yours into `/web/assets/frontend/images/custom_checkbox.png`
+    - download new images or add yours into `/web/assets/frontend/images/custom_checkbox.png` and `/web/assets/frontend/images/custom_radio.png`
```
Comment (s3tezsky) [inline upgrade/UPGRADE-v8.1.0-dev.md]: This one is reduntant to our first point. I guess we can remove this line.
```suggestion
```

File: upgrade/UPGRADE-v8.1.0-dev.md [hunk 4]
```diff
@@ -659,72 +663,80 @@ There you can find links to upgrade notes for other versions too.
...
+        +                {{ form_label(form, label, { label_attr: { class: "css-checkbox__image" }}) }}
+        +                {% set errors_attr = errors_attr|default({})|merge({'class': (errors_attr.class|default('form-error--choice'))}) %}
+        +                {{ form_errors(form, { errors_attr: errors_attr } ) }}
+        +                {{ block('icon') }}
+        +            </div>
+        +         </div>
+              {% endblock checkbox_row %}
+         ```
 
     - update all twig files according to pull request or find all checkboxes and add them `{ attr: { class: "css-checkbox" } }` and direct after input tag add label with `class="css-checkbox__image"`. You can change class name to `css-radio` and `css-radio__image` according to input type.
```
Comment (sslt) [inline upgrade/UPGRADE-v8.1.0-dev.md]: ...directLY after...
```suggestion
    - update all twig files according to pull request or find all checkboxes and add them `{ attr: { class: "css-checkbox" } }` and immediately after input tag add label with `class="css-checkbox__image"`. You can change class name to `css-radio` and `css-radio__image` according to input type.
```

**Pull request merged**